eureka-examples/build.gradle

```diff
@@ -3,6 +3,8 @@ apply plugin: 'application'
 dependencies {
     compile project(':eureka-client')
     compile files('conf')
+
+    compile 'org.slf4j:slf4j-simple:1.7.7'
```

Comment:
This is just for the example project so that we can actually see logs (cient dependency is on slf4j-api);